src/components/career-form/career-form.tsx

```diff
@@ -270,7 +270,7 @@ const CheckboxLabel = styled.label`
     content: '';
     display: inline-block;
     position: relative;
-    top: -3px;
```

Comment:
those magic numbers will bite us in the future, because we won't know if you read this from a design or if this came from some tinkering around values in the browser. I'm interested, where does this come from? I'm fine with the change of course here.